src/main/java/sirius/biz/analytics/charts/Dataset.java

```diff
@@ -63,6 +63,17 @@ public Dataset addValue(Number value) {
         return this;
     }
 
+    /**
+     * Adds the given values to the dataset
+     *
+     * @param values the values to add
+     * @return the dataset itself for fluent method calls
+     */
+    public Dataset addValues(List<Integer> values) {
```

Comment:
In the addValue method a Number is passed in, here it is an Integer. Shouldn't this be more consistent?